peepingtom/depictors/napari/particledepictor.py

```diff
@@ -90,3 +98,65 @@ def update(self):
             for layer, vector in zip(self.vectors, vectors):
                 layer.data = vector.values
                 layer.scale = self.datablock.pixel_size
+
+
+class ParticleMeshDepictor(ParticleDepictor):
+    """
+    depict particles as isosurface meshes based on an input volume
+    volume: if not an ImageBlock or a xarray, axis order must be zyx
+    """
+    def __init__(self, datablock, *, volume, iso=None, step_size=1):
+        super().__init__(datablock)
+        self.volume = volume
+        self.iso = iso
+        self.step_size = step_size
+        self.mesh_color = 'cornflowerblue'
+
+    def depict(self):
+        vertices, faces = self.make_volume_mesh()
+        all_vertices, all_faces = self.tile_mesh(vertices, faces)
+
+        self._make_surface_layer(all_vertices,
+                                 all_faces,
+                                 name=f'{self.name} - particles',
+                                 scale=self.datablock.pixel_size,
+                                 # TODO: color
+                                 )
+
+    def make_volume_mesh(self):
```

Comment:
`mesh_volume`?

Reply:
Can you use mesh as a verb in this context? o.o